feather/server/src/initial_handler.rs

```diff
@@ -58,6 +58,17 @@ pub async fn handle(worker: &mut Worker) -> anyhow::Result<InitialHandling> {
 
     let ClientHandshakePacket::Handshake(handshake) = handshake;
 
+    if handshake.protocol_version < PROTOCOL_VERSION {
```

Comment:
We should do this only if the next state is `Login`, because otherwise we disconnect status pings on older versions, which should still go through.